sourcegraph/sourcegraph
# [SG-39290] Accessibility Multiple H1 tags

## Description

There are multiple `<H1/>` elements: the Sourcegraph logo in the navigation bar, and the page title, which produces a warning in ARC Toolkit in the following pages:
1. [Analytics Page](https://k8s.sgdev.org/site-admin/analytics/search)
2. [Extensions Page](https://k8s.sgdev.org/extensions?category=All)
3. [Organization Settings Page](https://k8s.sgdev.org/organizations/sourcegraph/settings)
4. [Site Admin Page](https://k8s.sgdev.org/site-admin)

## Expected Behaviour
The page title should be an `<H2/>`, but retain the current style.
All other `<H2/>`s on the page should be downscaled to `<H3/>`, but retain their current styles.
No warnings are reported under ARC Toolkit's "Structure and Semantics" tests.

## Refs

- [Sourcegraph issue](https://github.com/sourcegraph/sourcegraph/issues/39290)
- [Gitstart ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-39290)

## Test plan
- Navigate to the pages in the `Description section`
- Ensure each page has only one `H1`and `H2` tag
- Ensure there is no UI change as a result of changes made

## App preview:

- [Web](https://sg-web-contractors-sg-39290.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-omjicfehul.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
